src/core/expr/head_reduce_unary.cc

```diff
@@ -1188,14 +1197,58 @@ per group::
      1 |     2      1
      2 |     3      1
     [3 rows x 2 columns]
+)";
+#endif
 
+template <typename T, typename U>
```

Comment:
why do you need `U` here?
Seems like you always have `U == T`.

Reply:
removed unwanted U.